### DISCUSSION TITLE
Help - Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by meteorquake** [sumatrapdfreader/sumatrapdf#3768] [Ideas]
My suggestion is a menu item -

* Help - Keyboard Shortcuts

which goes to https://www.sumatrapdfreader.org/docs/Keyboard-shortcuts
I know it's there via Help - Manual but it's probably the most regularly useful item for people to see and use and the menu only has 4 items on, and an easy thing to implement.
David

**Comment by kjk** [comment]
Done.
